packages/millicast-sdk/src/Publish.js

```diff
@@ -225,6 +230,22 @@ export default class Publish extends BaseWebRTC {
     promises = await Promise.all([getLocalSDPPromise, signalingConnectPromise])
     const localSdp = promises[0]
 
+    const worker = new Worker('workers/TransformWorker.js')
+
+    const senders = this.getRTCPeerConnection()
+      .getSenders()
+      .filter((elt) => elt.track.kind === 'video')
+    const sender = senders[0]
+
+    if (supportsRTCRtpScriptTransform) {
+      // eslint-disable-next-line no-undef
+      sender.transform = new RTCRtpScriptTransform(worker, { name: 'senderTransform' })
+    } else if (supportsInsertableStreams) {
+      const { readable, writable } = sender.createEncodedStreams()
+      worker.postMessage({ action: 'insertable-streams-sender', readable, writable }, [readable, writable])
```

Comment:
nitpick non-blocking : can we break this into multiple lines for easier readability please?